content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Python Flight client
+description: The Python Flight client integrates with Python scripts and applications to query data stored in InfluxDB.
+menu:
+  influxdb_cloud_dedicated:
+    name: Python
+    parent: Arrow Flight clients
+    identifier: python-flight-client
+influxdb/cloud-dedicated/tags: [Python, gRPC, SQL, Flight SQL, client libraries]
+aliases:
+  - /influxdb/cloud-dedicated/reference/client-libraries/flight-sql/python-flightsql/
+weight: 201
+list_code_example: |
+  ```py
+  from pyarrow.flight import FlightClient, Ticket, FlightCallOptions 
+  import json
+  import pandas
+  import tabulate
+
+  # Downsampling query groups data into 2-hour bins
+  sql="""
+    SELECT DATE_BIN(INTERVAL '2 hours',
+        time,
+        '1970-01-01T00:00:00Z'::TIMESTAMP) AS _time,
+      room,
+      selector_max(temp, time)['value'] AS 'max temp',
+      selector_min(temp, time)['value'] AS 'min temp',
+      avg(temp) AS 'average temp'
+    FROM home
+    GROUP BY
+      _time,
+      room
+    ORDER BY room, time"""
+    
+  flight_ticket = Ticket(json.dumps({
+    "namespace_name": "DATABASE_NAME",
+    "sql_query": sql,
+    "query_type": "sql"
+  }))
+
+  token = (b"authorization", bytes(f"Bearer DATABASE_TOKEN".encode('utf-8')))
+  options = FlightCallOptions(headers=[token])
+  client = FlightClient(f"grpc+tls://cluster-id.influxdb.io:443")
+
+  reader = client.do_get(flight_ticket, options)
+  arrow_table = reader.read_all()
+  ```
+---
+
+[Apache Arrow Python bindings](https://arrow.apache.org/docs/python/index.html) integrate with Python scripts and applications to query data stored in InfluxDB.
+
+The following examples show how to use the `pyarrow.flight` and `pandas` Python modules to query and format data stored in an {{% cloud-name %}} database:
+
+{{% code-tabs-wrapper %}}
+{{% code-tabs %}}
+[InfluxQL](#influxql-python)
+[SQL](#sql-python)
```

Comment:
For v3, I believe we are treating SQL as the primary query language, so SQL should probably be the first tab (default content).

Reply:
OK, thanks. I thought about that and then decided alphabetical.